inference-engine/src/mkldnn_plugin/emitters/jit_load_store_emitters.cpp

```diff
@@ -509,6 +509,12 @@ size_t jit_store_emitter::aux_vecs_count() const {
 
 size_t jit_store_emitter::get_inputs_num() const { return 1; }
 
+void jit_store_emitter::emit_data() const {
+    jit_emitter::emit_data();
```

Comment:
I think we can remove this function(jit_emitter::emit_data()) call that do noting, given no data need emitted for jit_emitter or jit_store_emitter itself currently as entry_map_ is empty, just need emit_data() for emu_vcvtneps2bf16.

Reply:
Is there no point at all to keep jit_emitter::emit_data()?
I mean I understand that currently it is kind of pointless, since map is empty, but in general, should we emit_data(), if map is not empty?

Reply:
suggest to remove from perf perspective because not all emitters need emit_data, if we really need, then we need explicitly register data_map and call emit_data() . I am ok as well because perf degradation is slight for this case.

Reply:
Done